packages/calling-component-bindings/src/utils/videoGalleryUtils.ts

```diff
@@ -53,6 +53,10 @@ export const _videoGalleryRemoteParticipantsMemo: (
             isPhoneNumberIdentifier(participant.identifier)
           );
         })
+        // Filter out MicrosoftBot participants
+        .filter((participant: RemoteParticipantState) => {
```

Comment:
will this also filter teams voice apps? I am worried that we are just filtering out all bots with this when we still need to show these other apps since they are the entity that the user will be calling in flows like c2c